lms/djangoapps/instructor/views/serializer.py

```diff
@@ -59,3 +59,40 @@ def validate_unique_student_identifier(self, value):
             return None
 
         return user
+
+
+class ListInstructorSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Comment:
Suggestion: I think a better name for this would be the `ListInstructorTaskInputSerializer`.